deeprankcore/DataSet.py

```diff
@@ -118,6 +125,32 @@ def __init__( # pylint: disable=too-many-arguments
             self.hdf5_path = [hdf5_path]
 
         self.target = target
+
+        # I made it so it overrules the user setting in case the target is one of the 'known' types.
+        # Is this what we want, or do we prefer to raise an error or show a warning in case it does not match? 
+        if self.target in ["irmsd", "lrmsd", "fnat", "dockQ"]:
+            self.task = "regress"
+        elif self.target in ["bin", "binary", "bin_class", "capri_classes"]:
+            self.task = "classif"
+        else:
+            self.task = task
```

Comment:
I made it so it overrules the user setting in case the target is one of the 'known' types. Is this what we want, or do we prefer to raise an error or show a warning in case it does not match?

Reply:
@gcroci2, did you see this?

Reply:
Yes it's fine :)